src/app/modules/form/public-form/public-form.controller.ts

```diff
@@ -219,70 +218,75 @@ export const handleGetPublicForm: ControllerHandler<
           return res.json({ form: publicForm, isIntranetUser })
         })
     case FormAuthType.MyInfo: {
+      // We always want to clear existing login cookies because we no longer
+      // have the prefilled data
+      res.clearCookie(MYINFO_LOGIN_COOKIE_NAME)
+      const authCodeCookie: unknown = req.cookies[MYINFO_AUTH_CODE_COOKIE_NAME]
+      // No auth code cookie because user is accessing the form before logging
+      // in
+      if (authCodeCookie === null || authCodeCookie === undefined) {
```

Comment:
nit: should we use a falsey check?

```suggestion
      if (!authCodeCookie) {
```

Reply:
can also!

Reply:
676072bc